Broken C#6 tutorial

The Focus mode editor throws the following compiler errors from page one:

(1,1): error CS8652:t The feature 'using declarations' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
(1,7): error CS0118: 'System' is a namespace but is used like a type
(1,13): error CS0210: You must provide an initializer in a fixed or using statement declaration
(1,13): error CS1001: Identifier expected
(1,1): error CS1513: } expected
(26,14): error CS0542: 'Program': member names cannot be the same as their enclosing type

Example code needs to be adjusted or focus mode editor needs to target the correct version.
The tutorial is functionally useless in its current state.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1a3686c2-4c73-5686-0da9-abc38c74aeec
* Version Independent ID: ca995f80-9375-e9ca-b855-6e6893581e23
* Content: [Explore C# 6 - Try the new features in C# 6 interactively, using your browser](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/exploration/csharp-6?tutorial-step=1)
* Content Source: [docs/csharp/tutorials/exploration/csharp-6.yml](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/exploration/csharp-6.yml)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/.openpublishing.redirection.json
+++ b/.openpublishing.redirection.json
@@ -2701,6 +2701,10 @@
             "redirect_url": "/dotnet/csharp/tutorials/default-interface-methods-versions"
         },
         {
+            "source_path": "docs/csharp/tutorials/exploration/csharp-6.yml",
+            "redirect_url": "/dotnet/csharp/whats-new/csharp-7"
+        },
+        {
             "source_path": "docs/csharp/tutorials/exploration/csharp-7.yml",
             "redirect_url": "/dotnet/csharp/whats-new/csharp-7"
         },

